revisionControl/forkingWorkflow/text.md

```diff
@@ -9,19 +9,19 @@
 <img src="{{baseUrl}}/revisionControl/forkingWorkflow/images/diagram.png" height="330" />
 <p/>
 
-**In the _forking workflow_, the 'official' version of the software is kept in a remote repo designated as the 'main repo'. All team members fork the main repo create pull requests from their fork to the main repo.**
+**In the _forking workflow_, the 'official' version of the software is kept in a remote repo designated as the 'main repo'. All team members fork the main repo and create pull requests from their fork to the main repo.**
 
 To illustrate how the workflow goes, let’s assume Jean wants to fix a bug in the code. Here are the steps:
 
 1. Jean creates a separate branch in her local repo and fixes the bug in that branch.<br>
-   {{ icon_important_big_red }} Common mistake: Done the proposed changes in the `master` branch -- if Jean does that, she will not be able to have more than one open PR at any time because any changes to the `master` branch will be reflected in all open PRs.
+   {{ icon_important_big_red }} Common mistake: Do the proposed changes in the `master` branch -- if Jean does that, she will not be able to have more than one open PR at any time because any changes to the `master` branch will be reflected in all open PRs.
```

Comment:
```suggestion
   {{ icon_important_big_red }} Common mistake: Doing the proposed changes in the `master` branch -- if Jean does that, she will not be able to have more than one open PR at any time because any changes to the `master` branch will be reflected in all open PRs.
```